<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionDigital_phosphor"/>
    <addaction name="actionZoom"/>
    <addaction name="actionManualCommand"/>
   </widget>
   <widget class="QMenu" name="menuOscilloscope">
    <property name="title">
     <string>&amp;Oscilloscope</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionSampling"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuOscilloscope"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="separator"/>
   <addaction name="actionPrint"/>
   <addaction name="actionExport"/>
   <addaction name="separator"/>
   <addaction name="actionSampling"/>
   <addaction name="separator"/>
   <addaction name="actionDigital_phosphor"/>
   <addaction name="actionZoom"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../res/application.qrc">
     <normaloff>:/actions/open.png</normaloff>:/actions/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../res/application.qrc">
     <normaloff>:/actions/save.png</normaloff>:/actions/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="../res/application.qrc">
     <normaloff>:/actions/save-as.png</normaloff>:/actions/save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save as ...</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../res/application.qrc">
     <normaloff>:/actions/print.png</normaloff>:/actions/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="../res/application.qrc">
     <normaloff>:/actions/export-as.png</normaloff>:/actions/export-as.png</iconset>
   </property>
   <property name="text">
    <string>Export as ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDigital_phosphor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/application.qrc">
     <normaloff>:/actions/digitalphosphor.png</normaloff>:/actions/digitalphosphor.png</iconset>
   </property>
   <property name="text">
    <string>Digital phosphor</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/application.qrc">
     <normaloff>:/actions/zoom.png</normaloff>:/actions/zoom.png</iconset>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
  </action>
  <action name="actionDocking_windows">
   <property name="text">
    <string>Docking windows</string>
   </property>
  </action>
  <action name="actionToolbars">
   <property name="text">
    <string>Toolbars</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSampling">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/application.qrc">
     <normaloff>:/actions/stop.png</normaloff>
     <normalon>:/actions/start.png</normalon>:/actions/stop.png</iconset>
   </property>
   <property name="text">
    <string>Sampling</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionManualCommand">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Manual command</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/application.qrc"/>
 </resources>
 <connections/>
</ui>
